### ISSUE TITLE
Extension to Demographic and Biographic data points in 2 types of entity

### ISSUE THREAD

**Issue by cpoma** [LuatixHQ/opencti#2661]
## Use case

This feature is requesting the addition of some extended characteristic with regard to Identity (sub-type: Individual). It would the ability to track Demographic and Biographic data should the Identity be able to be isolated an identified as a real world "person" / individual.

The Demographic data to track would be:

-  **Country of Citizenship** - [Array of Strings]. Ideally, this would link to a list of enums/lookups of official ISO 3166 identified Countries that can be selected
-  **Country of Residence** - [Array of STIX Location objects].
-  **Date of Birth** - Date field
-  **Place of Birth** - STIX Location or String - could be implemented either way - however - STIX Location seems more feature rich.
-  **Employer** - [Array of STIX Location] or [Array of Strings] - could be implemented either way - however - Array of Location seems more feature rich.
-  **Ethnicity** - [Array of Strings] - Ideally, this would link to a list of enums/lookups of official ISO identified ethnicities that can be selected
-  **Gender** - [Array of Strings] - Ideally, this would link to a list of enums/lookups of official ISO identified genders that can be selected
-  **Marital Status** - [Array of Objects] - The objects would have (Start Date(Optional), End Date(Optional), String (Value from enums/lookups of official ISO identified status that can be selected - Married, Divorced, Widowed... etc) 
- **Nationality** - [Array of Strings] - Ideally, this would link to a list of enums/lookups of official ISO identified nationalities that can be selected

The Biographic data to to track would be:

- **Eye Color** - [Array of Strings] - This would link to a list of enums/lookups; Users should have ability to add to this enum/lookup list new colors
- **Hair Color** - [Array of Strings] - This would link to a list of enums/lookups; Users should have ability to add to this enum/lookup list new colors
- **Weight_KG** - [Float] - Input in this field will convert and update the Weight_LB to match via conversion
- **Weight_LB** - [Float] - Input in this field will convert and update the Weight_KG to match via conversion
- **Height_M** - [Float] - Input in this field will convert and update the Weight_IN to match via conversion
- **Height_IN** - [Float] - Input in this field will convert and update the Weight_M to match via conversion

## Current Workaround

There is currently no capability to store these fields within OpenCTI. They are not part of the standard STIX 2.1 schema.

## Proposed Solution

Implementation of these fields using STIX Extensions. There are three ways to extend STIX using STIX Extensions.

- Define one or more new STIX Object types.
- Define additional properties for an existing STIX Object type as a nested property extension. This is typically done to represent a sub-component or module of one or more STIX Object types.
- Define additional properties for an existing STIX Object type at the object's top-level. This can be done to represent properties that form an inherent part of the definition of an object type.

The developer should chose the method they find most appropriate and capable of incorporation into OpenCTI's data model/schema.

## Additional Information

None

## If the feature request is approved, would you be willing to submit a PR?

Yes / No (Help can be provided if you need assistance submitting a PR)

This Feature Request is currently structured with the expectation the OpenCTI developer would be doing the implementation. However, Yes - if assistance is needed, it could provided by outside parties.

**Comment by SamuelHassine** [comment]
Blocked by #3166.

This feature should be implemented for both Threat Actor Individual and Identity Individual.

**Comment by SamuelHassine** [comment]
![image](https://user-images.githubusercontent.com/1334279/232197439-a1e669bf-5092-4e79-ae1f-162719013520.png)

**Comment by ParamConstructor** [comment]
@richard-julien / @SamuelHassine / @Jipegien - Recommend Closure. This was released in 5.11.
